packages/elements-dev-portal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stoplight/elements-dev-portal",
-  "version": "2.4.4",
+  "version": "2.4.5",
```

Comment:
You also need to upgrade `elements-core` version in the dependencies here below and in the `packages/elements/package.json`.